src/AppBundle/Resources/config/remote_media.yml

```diff
@@ -0,0 +1,70 @@
+system:
+    default:
+        image_variations:
+            default:
+                small:
```

Comment:
Shouldn't `small`, `medium` and `large` go only to `ngadmin_group`, like in eZ image variation config https://github.com/netgen/media-site/blob/1.10/src/AppBundle/Resources/config/image.yml#L41?

Reply:
These three also shouldn't have crop.

Reply:
Moved in 794f229b